Cargo.toml

```diff
@@ -13,12 +12,14 @@ An implementation of regular expressions for Rust.
 """
 
 [[test]]
-path = "tests/mod.rs"
+path = "regex_macros/tests/test_dynamic.rs"
+name = "all"
+
+[[bench]]
 name = "all"
+path = "regex_macros/tests/bench_dynamic.rs"
```

Comment:
Perhaps this benchmark could be placed in the `benches` directory instead of the `tests` directory?